Gordon360/Models/ViewModels/RecIM/SportViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Gordon360.Models.CCT;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class SportViewModel
+    {
+        public int ID { get; set; }
+        public string? Name { get; set; }
+        public string? Description { get; set; }
+        public string? Rules { get; set; }
+        public string? Logo { get; set; }
```

Comment:
Are these all nullable in the database? If not, why are they nullable here?

I would think Name should at least always have a value, otherwise, what is the point of a sport?

Reply:
The nullables will be moved to a SportPatchViewModel as you suggested, then the normal SportViewModel will have non-nullable fields